.gitignore

```diff
@@ -35,6 +35,7 @@ perf.data*
 /scratch
 services.log
 *.swp
+/temp
```

Comment:
Why not just use python's https://docs.python.org/3/library/tempfile.html instead btw?

Reply:
I had a look at it.

I don't want the files to be that temporary 😂. They should survive the termination of the program so that another invocation can make use of `--no-fetch`.